documentation/asciidoc/topics/proc_adding_credentials_keystore.adoc

```diff
@@ -0,0 +1,24 @@
+[id='adding-credentials_keystore_{context}']
+= Adding credentials keystore
+
+[role="_abstract"]
+Create {brandname} credentials keystore to avoid
+writing password in clear in the server config file. See <<Enabling TLS encryption>> for a use case.
```

Comment:
"to avoid exposing passwords in clear text in the server configuration ConfigMap"?